extensions/wikia/EditPageLayout/EditPageLayoutController.class.php

```diff
@@ -173,7 +173,7 @@ public function executeEditPage() {
 		$this->notificationsLink =
 			( count( $this->notices ) == 0 )
 			? wfMessage( 'editpagelayout-notificationsLink-none' )->escaped()
-			: wfMessage( 'editpagelayout-notificationsLink', count( $this->notices ) )->escaped();
+			: wfMessage( 'editpagelayout-notificationsLink', count( $this->notices ) )->text();
```

Comment:
This should be `->parse()` not `->text()`. It's still being output as HTML so `->text()` is not a safe output format, and `<span>` is valid wikitext, so is fine.